### PR TITLE
completed es6 operators

### PR DESCRIPTION
**Pull request by AaronHe09**
image

### CODE REVIEW

File: es6-operators/main.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+const v4 = [3, 5, 7, 9];
+
+/* TODO:
+ * Demonstrate how to execute an expression only if a variable is truthy.
+ * In one expression, log "v1 is truthy" to the console only if `v1` is truthy.
+ * In one expression, log "v2 is truthy" to the console only if `v2` is truthy.
+ * Expected: v1 message logs, v2 message does not.
+ */
+console.log(v1 && 'v1 is truthy');
+console.log(v2 && 'v2 is truthy');
```
Comment (shawnkost) [inline es6-operators/main.js:14]: We should only be logging if the first value is truthy. Right now you are logging regardless, your 2nd console log just isn't printing any value

**Review by shawnkost — CHANGES_REQUESTED** [review]
One change here Aaron!

File: es6-operators/main.js (hunk 1 above)
Comment (AaronHe09) [reply es6-operators/main.js]: I think i got it now?
v1 && console.log('v1 is truthy');
v2 && console.log('v2 is truthy');

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**